### PR TITLE
Fix star popup scaling and star null ref

### PR DESCRIPTION
**Pull request by mklewandowski**

### CODE REVIEW

File: Assets/Scripts/StarComponent.cs [hunk 1]
```diff
@@ -17,7 +17,10 @@ public class StarComponent : MonoBehaviour
     void Start()
     {
         dataControllerObj = GameObject.Find("DataController");
-        dataController = dataControllerObj.GetComponent<DataController>();
+        if (dataControllerObj)
+        {
+            dataController = dataControllerObj.GetComponent<DataController>();
```
Comment (dstrawberrygirl) [inline Assets/Scripts/StarComponent.cs]: I didn't think truthy-if worked in C#? Regardless, why not simplify this:
`
if (dataController == null) dataController = FindObjectOfType<DataController>();
`

Comment (mklewandowski) [reply Assets/Scripts/StarComponent.cs]: Yeah, that works.  You can just test for the object without a comparison to null.

Comment (dstrawberrygirl) [reply Assets/Scripts/StarComponent.cs]: good to know! In which case, let's use:
`if (!dataController) dataController = FindObjectOfType<DataController>();`

Comment (mklewandowski) [reply Assets/Scripts/StarComponent.cs]: But basically I'm saying only if we can find the game object named DataController, do we try to get a ref to the script on the object.  So I'm talking about both the object and the script on the object.

**Pull request merged**